companion/src/simulation/simulatorinterface.cpp

```diff
@@ -59,7 +59,7 @@ void registerSimulators()
   QStringList filters;
 #if defined(__APPLE__)
   filters << "*-simulator.dylib";
-#elif (!defined __GNUC__) || (defined __CYGWIN__)
```

Comment:
This breaks the simulator on Windows running under Cygwin (posix).  It still compiles but fails to register the dll.
Changing #elif (defined WIN32) || (defined __CYGWIN__) fixes it.